docs/basic-docs/atomicdex-api-20/trade_preimage.md

```diff
@@ -47,12 +47,13 @@ Use the `trade_preimage` request with `max = true` and `swap_method = "setprice"
 
 Where the `ExtendedFeeInfo` has
 
-| Structure       | Type             | Description                                                                                                                                                   |
-| --------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| coin            | string           | the fee is paid from the user's balance of this coin. This coin name may differ from the `base` or `rel` coins. For example, ERC20 fees are paid by ETH (gas) |
-| amount          | string (numeric) | fee amount (in decimal representation)                                                                                                                        |
-| amount_rat      | rational         | fee amount (in rational representation)                                                                                                                       |
-| amount_fraction | fraction         | fee amount (in fraction representation)                                                                                                                       |
+| Structure             | Type             | Description                                                                                                                                                   |
+| --------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| coin                  | string           | the fee is paid from the user's balance of this coin. This coin name may differ from the `base` or `rel` coins. For example, ERC20 fees are paid by ETH (gas) |
+| amount                | string (numeric) | fee amount (in decimal representation)                                                                                                                        |
+| amount_rat            | rational         | fee amount (in rational representation)                                                                                                                       |
+| amount_fraction       | fraction         | fee amount (in fraction representation)                                                                                                                       |
+| paid_from_trading_vol | boolean          |  If `true`, fees are deducted from the payment amount for the spend/refund UTXO HTLC transaction. If `false`, fees are not deducted from the traded volume. This is either where maker side pays additional miner fee to broadcast their swap transaction, or where gas paid (e.g in ETH for an ERC20 trade) - in this case, user requires a sufficient current ETH balance to cover the fees before they can initiate the swap)                                             |
```

Comment:
> maker side pays additional miner fee

I used "maker" as an example - it applies to taker transactions (dex fee, swap payment) too. Could you change it please to reflect this meaning?

Reply:
reworded, thanks.